testsuite/features/step_definitions/cobbler_steps.rb

```diff
@@ -234,3 +233,16 @@
   out, _code = $server.run('cobbler sync')
   raise 'cobbler sync failed' if out.include? 'Push failed'
 end
+
+When(/^I start local monitoring of Cobbler$/) do
+  $server.run('tail -n 0 -f /var/log/cobbler/cobbler.log > /var/log/cobbler/testsuite_cobbler_watch.log', timeout: 1)
+end
+
+When(/^I check for Cobbler errors in the local logs$/) do
+  command_output, _code = $server.run('ps axo pid,cmd | grep tail | grep -v grep', check_errors: false)
+  pid = command_output.split(' ')[0]
+  $server.run("kill #{pid}", check_errors: false)
+  output, code = $server.run('grep -i error /var/log/cobbler/testsuite_cobbler_watch.log', check_errors: false)
+  $server.run('rm /var/log/cobbler/testsuite_cobbler_watch.log', check_errors: false)
+  raise "Errors in Cobbler logs:\n #{output}" if code.zero?
+end
```

Comment:
That is a suboptimal solution. You can configure the logging of Python applications with a `logging.conf`. This is also valid for Cobbler and would make the logfiles machine readable. Just grepping for errors will not help since you don't have all metadata for template rendering in most cases. So you will have a lot of false-positives that need to be filtered out.

Reply:
Please consult @nodeg for this as he is working on improvements in parallel already, as well as having the knowledge to work with the logging of Cobbler.

Reply:
If we do so, I would vote to integrate it as an optional parameter in sumaform. So in the environments we need, we use a parameter that set the logging for Cobbler as we need.

Reply:
But ofc, all of that, in my humble opinion, can be done in future iterations of these changes.

Reply:
My concern was addressed and thus I resolve this conversation.